purchases/src/test/java/com/revenuecat/purchases/google/attribution/GoogleDeviceIdentifiersFetcherTests.kt

```diff
@@ -73,7 +73,7 @@ class GoogleDeviceIdentifiersFetcherTests {
             mockContext = mockContext,
             expectedAdID = "12345",
             expectedAndroidID = "androidid",
-            expectedException = GooglePlayServicesRepairableException(1, "error", null)
+            expectedException = GooglePlayServicesRepairableException(1, "error", mockk())
```

Comment:
Looks like Google changed this from being nullable to non-nullable in 6.1.0, which is a breaking change 😬...

Reply:
👏🏻